readme.md

```diff
@@ -83,6 +83,22 @@ Works in synchronous or asynchronous mode.
     }, 
 ```
 
+#### Waiting for an async process to be ready:
+
+Wait for a background process to signal that it's ready before continuing on to the next task by providing `readyRegex`.
+
+```
+    proxy: {
+        command: 'browsermob-proxy',
+        options: {
+            async: true,
+            readyRegex: /Started SelectChannelConnector/
+        }
+    },
+```
+
+This is useful for making sure a proxy is ready to receive requests before beginning tests.
```

Comment:
This would apply to a listening server as well.  I like the idea.  This is an issue Apache Thrift unit tests have had in various languages.